_posts/2020-10-20-3.8.0-release-announcement.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Pulp 3.8.0 is Generally Available!
+author: Melanie Corr
+tags:
+  - announcement
+---
+
+The Pulp team are pleased to announce the release of Pulp 3.8.0!
+
+## Updates to Django settings
+
+* The Django `admin` site URL is now configurable using the new `ADMIN_SITE_URL` settings parameter [7637](https://pulp.plan.io/issues/7637).
+
+* As a follow up to [#7720](https://pulp.plan.io/issues/7720), which provided a default `DJANGO_SETTINGS_MODULE` variable, in this release, the default `DJANGO_SETTINGS_MODULE` variable is now set everywhere. This simplifies configuration because the **systemd** service no longer has to set this value.
```

Comment:
Did you mean to mention here the #7179? #7720 is a part of the 3.8.0 release.

Reply:
Copy paste. Thank you!